### PR TITLE
Add entry ember europe london

### PR DESCRIPTION
**Pull request by emmasofiah2o**
Screenshot 2022-11-07 at 10 23 44

### CODE REVIEW

**Review by marcoow — CHANGES_REQUESTED** [review]
the SVG is slightly broken but I think we have a PNG as well

**Comment by emmasofiah2o** [comment]
> the SVG is slightly broken but I think we have a PNG as well

Just changed, let me know if this works. I wasn't sure if I was supposed to use the logo with the big ben or the general one :).

**Review by marcoow — CHANGES_REQUESTED** [review]
I don't think this image works well tbh…

Bildschirm­foto 2022-10-31 um 12 34 30

**Comment by emmasofiah2o** [comment]
> I don't think this image works well tbh…
> 
> Bildschirm­foto 2022-10-31 um 12 34 30

Is it because it misses the background? We only have transparent logos of Ember Europe, but I can create an extra one with a background if you like

**Comment by marcoow** [comment]
I'd say it looks broken because it's apparently cut off and not meant for sth. like this? I don't think a background would make that much better. Maybe just the flags would work for example – be creative, I'm sure you'll find something that works well

**Comment by emmasofiah2o** [comment]
> I'd say it looks broken because it's apparently cut off and not meant for sth. like this? I don't think a background would make that much better. Maybe just the flags would work for example – be creative, I'm sure you'll find something that works well

I thought it looked fine tbh :D. I'll see if I can come up with smth

**Comment by emmasofiah2o** [comment]
Ok I figured out the problem. I thought the screenshot you sent was just weirdly cut - this one should work :) @marcoow

**Comment by marcoow** [comment]
> I thought the screenshot you sent was just weirdly cut

always make sure to check out the changes on the preview system yourself before asking for review 🙏

**Review by marcoow — CHANGES_REQUESTED** [review]
I think it's fine but extremely blurry. Also it shouldn't be an SVG as it contains binary image data – use PNG instead but make sure to use a larger resolution

**Comment by emmasofiah2o** [comment]
@marcoow I have tried changing it again, it is slightly bigger but still blurry. The problem is that I need to use Tomster's t-shirt to make the background cause I don't have an SVG of the background on its own. And my computer crashes if I try to make it bigger than how it is now...

**Comment by marcoow** [comment]
please ask @davidhajba for help

**Comment by davidhajba** [comment]
Hey Marco, 
Emma asked me to explain the situation from my perspective:
The picture both JPEG and PNG should work perfectly in this size that I sent to Emma. 
As it is distorted, it is not a visual design, but a development issue.
Maybe the website reduces file size significantly or there’s some error displaying the image.

**Review by marcoow — APPROVED** [review]

File: src/calendar/2022-12-08-embereurope-london.md [hunk 1]
```diff
@@ -0,0 +1,12 @@
...
+image: "/assets/images/calendar/2022-12-08-embereurope-london/embereurope.png"
+location: London & remote
+url: https://www.meetup.com/ember-europe/
+kind: meetup
+color: aqua
+---
+
+The Europe Ember.js meetup is a one-day event, bringing together the Ember
+community from all of Europe once a quarter. The meetup is organized as a hybrid
+event to allow people to join remotely as well.
```
Comment (marcoow) [inline src/calendar/2022-12-08-embereurope-london.md]:
```suggestion
Ember Europe series is back in Q4 with the last event for the year, this time hosted by CrowdStrike in London. The Ember Europe meetup series brings together the Ember community from all of Europe once a quarter. The meetup is organized by Mainmatter and Intercom as a hybrid event, hosted from a different city each time with the option for people to join remotely as well.
```

**Review by marcoow — CHANGES_REQUESTED** [review]
it's not a 1 day event but a regular evening meetup

**Comment by marcoow** [comment]
@emmasofiah2o for any PR that results in visual changes (which is essentially every PR to the website), it's always helpful to add a screenshot in the description to give people a quick glance at what's changing

**Comment by emmasofiah2o** [comment]
@marcoow I have committed your suggestion. As for the background, it is still blurry (as David explained, this might be a different type of issue). Let me know if I can merge. I also added the screenshot in the description of the pr as you recommended.

File: src/calendar/2022-12-08-embereurope-london.md [hunk 2]
```diff
@@ -0,0 +1,14 @@
+---
+title: "Ember Europe"
+image: "/assets/images/calendar/2022-12-08-embereurope-london/embereurope.png"
+location: London & remote
+url: https://www.meetup.com/ember-europe/
+kind: meetup
+color: aqua
+---
+
+Ember Europe series is back in Q4 with the last event for the year, this time
```
Comment (marcoow) [inline src/calendar/2022-12-08-embereurope-london.md]:
```suggestion
Ember Europe is back in Q4 with the last event for the year, this time
```

If using series I think it requires "The " as well but that's how the next sentence starts

File: src/calendar/2022-12-08-embereurope-london.md [hunk 3]
```diff
@@ -0,0 +1,14 @@
+---
+title: "Ember Europe"
```
Comment (marcoow) [inline src/calendar/2022-12-08-embereurope-london.md]: Let's also use a better title

```suggestion
title: "Ember Europe Q4 Meetup"
```

File: src/calendar/2022-12-08-embereurope-london.md (hunk 2 above)
Comment (emmasofiah2o) [reply src/calendar/2022-12-08-embereurope-london.md]: You wrote the text, I just committed your suggestion :D. I'll change it

**Review by marcoow — APPROVED** [review]

**Comment by emmasofiah2o** [comment]
Can I merge this @marcoow?

**Comment by marcoow** [comment]
yes, it's approved – you can merge without asking again

**Pull request merged**